addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -22,6 +22,7 @@ signal background_event_finished(return_code, event_name, channel_name)
 
 # Pre-defined ESC events
 const EVENT_DEBUG = "debug"
```

Comment:
If we're removing `debug` all together, can we remove this constant? Or is it used elsewhere?